src/StatusTable/__tests__/useStatusTableUtil.test.js

```diff
@@ -0,0 +1,112 @@
+import { filterData, getCompareFormat, mapTableFilters, sortData } from '../useStatusTableUtil';
+
+const FILTER_VALUES = {
+    logId: 'id123',
+    dateModified: '2020-01-20'
+};
+
+const DATA = [
+    {
+        log_id: 't408/data-01-20-2021_13-11-10.tlog',
```

Comment:
Could we get rid of more of the date / time information as well?  If you want to keep the tests the same, you could leave the very last segment -- e.g., `t408/data-10.tlog`.

Reply:
removed the date from the links.